check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
+    if (@exclude){
+        foreach my $line (@rs){
+	    foreach my $exclude_re (@exclude){
```

Comment:
I would rather move this loop directly in the beginning of the loop at line 5587 and replace the `delete` by te following code:

```
next if grep { $_ =~ $exclude_re } @{ $host }
```

At least, it would follow the same template code than other services using --exclude.

Thoughts?